lessons/lesson5.md

```diff
@@ -0,0 +1,234 @@
+## Lesson 5: Add Personalized Promotion Emails Triggering
+
+*Note: This is an advanced part of the codelab. You are recommended to go through it, but feel free to skip and jump to the [Recap](recap.md).*
```

Comment:
I think you're right this should be a different codelab ie an extension to this one.